### PR TITLE
Assignment 2b - z170037

### PR DESCRIPTION
**Pull request by adrix1341**

### CODE REVIEW

File: Assignment2/adrix1341-fromRDFtoJSON-LD.jsonld [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    },
+    {
+      "@id": "pe:User10A",
+      "pr:hasName": {
+        "@value": "Pedro",
+        "@type": "xsd:string"
+      }
+    }
+  ]
+}
```
Comment (paoespinozarias) [inline Assignment2/adrix1341-fromRDFtoJSON-LD.jsonld:49]: It is ok, good job!

File: Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld [hunk 2]
```diff
@@ -0,0 +1,61 @@
+{
+	"@context":{
+		"xsd": "http://www.w3.org/2001/XMLSchema#",
+		"person":"http://www.ontologies.org/ontologies/People#"
```
Comment (paoespinozarias) [inline Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld:4]: You forgot defining the attributes.

File: Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+			"@value":"Eliza",
+			"@type":"xsd:string"
+		}
+	},
+	"subjects":[{
+		"name":{
+			"@value":"Linked Data",
+			"@type":"xsd:string"
+		},
+		"Teachers":{
```
Comment (paoespinozarias) [inline Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld:36]: Be careful with the way you defined teachers' values.

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld (hunk 2 above)
Comment (paoespinozarias) [reply Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld:4]: Still pending to add the attribute definitions into "@context" or add the corresponding prefix to them below

**Review by paoespinozarias — CHANGES_REQUESTED** [review]

File: Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld (hunk 2 above)
Comment (adrix1341) [reply Assignment2/adrix1341-fromJSONtoJSON-LD.jsonld]: I just made a change in the file. Sorry for taking time to answer

**Review by paoespinozarias — APPROVED** [review]

**Pull request merged**